src/onegov/election_day/assets/js/embedded_widgets.js

```diff
@@ -0,0 +1,15 @@
+var createEmbedLink = function(el)  {
+    var embedLink = $(el).data('embed-link');
+    var embedSource = $(el).data('embed-source');
+    if (embedLink && embedSource) {
```

Comment:
I would think that if you run `eslint` on this, you would find some errors. We haven't set this up on our CI yet, but I would strongly encourage you run this file (and all your javascript changes) against eslint. There's an `.eslintrc` in the repository root that should get picked up to tell you what rules you should follow.